doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procure for creating a Fluent Docker container uses a local
+Ubuntu machine to copy the needed files from the Fluent
+
+
+Requirements
+============
+
+* A Linux machine, preferably with Ubuntu 18.04 or later.
```

Comment:
_preferably_ is a loose term when listing requirements/prereqisites. What are the consequences of choosing a different version? Are there any hard limits? If we aren't aware of the limits then we can explicitly recommend Ubuntu 18.04 and say that we have only validated the procedure for Ubuntu 18.04 and we have not validated the procedure for any other distributions .

Reply:
We just need to copy files. Any Linux version is suitable. Done. Thank you.